src/api-service/__app__/onefuzzlib/repro.py

```diff
@@ -86,6 +87,20 @@ def init(self) -> None:
                 self.state = VmState.extensions_launch
         else:
             result = vm.create()
+            nsg = NSG(
+                name=vm.region,
+                region=vm.region,
+            )
+            result = nsg.create()
+            if isinstance(result, Error):
+                self.set_failed(result)
+                return
+
+            result = nsg.set_allowed_sources(["*"])
```

Comment:
Apply earlier comment about documenting, and incoming IP defaults. Don't we actually want to ensure that we reject all incoming sources, by default?